manageiq-providers-cisco_intersight.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "intersight_client"
+  spec.add_dependency "intersight_client", ">= 0.1.4"
```

Comment:
@jrafanie what do you think of doing `~>0.1` so we can pick up minor version bumps without also pulling in breaking 1.0 changes?  Maybe `"~> 0.1", ">= 0.1.4"` if you want to require 0.1.4 at a minimum?

Reply:
I'm fine with that. I didn't have opinions on it.  I'll update it.